lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -280,11 +347,125 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         this.categoryControlVisible = false;
     }
 
-    cancelChanges() {
+    cancelChanges(buttonType: ButtonType, event: MouseEvent, group?: CardViewGroup) {
+        event.stopPropagation();
+        this.toggleEditMode(buttonType, group);
         this.revertChanges();
         this.loadProperties(this.node);
     }
 
+    isEditingPanel(): boolean {
```

Comment:
add code comments/docs for public functions, i.e.

```
// Returns the editing state of the panel
isEditingPanel(): boolean
```

Reply:
I have updated the comments